### PR TITLE
[POWEROPS-2225] Changing containers

### PR DESCRIPTION
**Pull request by doctrino**
## Description
Please describe the change you have made.

## Checklist:
- [ ] Tests added/updated.
- [ ] Documentation updated.
- [ ] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/power-ops-sdk/blob/master/CHANGELOG.md).
- [ ] Version bumped. If triggering a new release is desired, bump the version number in [pyproject.toml](https://github.com/cognitedata/power-ops-sdk/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -68,8 +68,12 @@ tomli = {version = "^2.0.1", python = "<3.11"}
...
-cognite-toolkit = {version="0.1.0b9"}
+# As of 29/02-24 there is a bug in the toolkit that prevents us from deleting containers.
+# This is fixed in the installation-branch. However, we cannot release power-ops-sdk with a dependency on the installation-branch.
+cognite-toolkit = {git="https://github.com/cognitedata/cdf-project-templates.git", branch="installation-branch"}
+# Therefore, you must install this version of the toolkit after deleting the container, such that
+# the power-ops-sdk can be released with a stable version of the toolkit.
+#cognite-toolkit = {version="0.1.0"}
 cognite-pygen = {version = ">=0.99.10", extras = ["all"]}
 
 mkdocs =  {version="*", optional=true}
```
Comment (eriklien) [inline pyproject.toml:79]: Why is a most of this appearing with red highlighting/background? Should mean syntax error, I think, but...

**Review by eriklien — APPROVED** [review]
Changing containers seems quite hard...
How much effort would it be to improve this (e. g. enabling deletion of a single container) - in the toolkit or elsewhere?
And how long do you expect that the stable/released version of cognite-toolkit can delete containers again?

I guess we need to do it like this for now to keep progress, so I approve it.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -35,3 +36,39 @@ To change a data model, follow these steps:
...
 9. Update the version in `pyproject.toml` and `cognite/powerops/_version.py` to the next version. Do a patch version bump.
 10. Get the PR approved and merge it.
+
+
+### Changing Containers
+Changing the containers requires dropping existing containers and recereating the changed ones. This is currently
+not supported by the `powerops` CLI, so you will have to do it using `cdf-tk`, i.e., `cognite-toolkit`, directly.
+
+**Note** There is currently no support for only dropping a single container, you will have to drop every container
+in the data model(s) and recreate them.
```
[the remark this answers is not in the document]
Comment (doctrino) [reply CONTRIBUTING.md:46]: All containers in v1 of the model.

File: pyproject.toml (hunk 1 above)
Comment (doctrino) [reply pyproject.toml:79]: No idea.

**Comment by doctrino** [comment]
> Changing containers seems quite hard... How much effort would it be to improve this (e. g. enabling deletion of a single container) - in the toolkit or elsewhere? And how long do you expect that the stable/released version of cognite-toolkit can delete containers again?
> 
> I guess we need to do it like this for now to keep progress, so I approve it.

Stable version will be latest early next week. 

Doing more surgical deletion of containers is not on the roadmap for the toolkit at this stage, but I might add it. Problem is that I would have to implement it, and I do not have time to do it.

**Pull request merged**